### DISCUSSION TITLE
Adding a scrape error runtime metric?

### DISCUSSION THREAD

**Discussion by salehabbas** [tomkerkhove/promitor#1635] [Ideas] [answered]
Greetings, 

Just was thinking if enriching the promitor _runtime metrics_ by adding something like `promitor_scrape_error` that reports any _scrapping errors_ happening would be a good idea? Just was thinking about it while I was setting some monitoring/alerting for the Promitor pods themselves :)

**Comment by tomkerkhove** [comment, marked as answer]
I'm tracking this on https://github.com/tomkerkhove/promitor/issues/1304 so feel free to have a look and provide feedback!

**Reply by salehabbas** [reply]
Thanks Tom! Will do.
